libs/shared/src/components/CopyToClipboard/index.tsx

```diff
@@ -9,15 +9,18 @@ interface Props {
   label?: MessageDescriptor;
 }
 
-const translate = defineMessages({ 
-    link : { 
-      id: "base.copyToClipboard.link",
-      defaultMessage:"Copy",
-      description:"base.copyToClipboard.link: Link for the button for the copy to clipboard tool"},
-    messageSuccess : { 
-      id: "base.copyToClipboard.messageSuccess",
-      defaultMessage:"Copied to your clipboard",
-      description:"base.copyToClipboard.messageSuccess: Success message for the button for the copy to clipboard tool"}});
+const translate = defineMessages({
+  link: {
+    id: "base.copyToClipboard.link",
```

Comment:
Is it possible to remove those ids?

Reply:
Eeek - ah, nuts.   I see that's in your site PR that I'm working on now.  Hopefully git can figure out the merge.  I'll merge that PR into here as soon as it's complete (I added a check to make sure no committed files are changed on build - basically to check that any translation changes are managed on developer end.  However, it seems to be failing on unrelated issues, and I might have to remove it)